packages/widget/src/components/transfer/fungible/fungible-token-transfer.ts

```diff
@@ -142,7 +161,7 @@ export class FungibleTokenTransfer extends BaseComponent {
       </section>
       <section>
         <sygma-address-input
-          .address=${this.transferController.destinatonAddress}
+          .address=${this.getSenderDefaultDestinationAddress()}
```

Comment:
This isn't triggering an update in transferController. So I've ended up in this state:
![image](https://github.com/sygmaprotocol/sygma-widget/assets/8836210/fd534ce7-92bb-4593-9a03-27c15b04ba9f)

Reply:
@Lykhoyda maybe it'd make more sense to put all that logic in `transferController` instead of having it inside a component? also, that'd would probably autofix this bug with not updating state :)

Reply:
@sztok7 good idea! Take a look please if I miss something